client/src/app/core/definitions/action/assignment-action.ts

```diff
@@ -0,0 +1,42 @@
+import { HasMeetingId } from 'app/shared/models/base/has-meeting-id';
```

Comment:
General structure: The `definitions`-folder is some kind of misc-thing. I would prefer the actions to be more prominent: Move the `action` folder one layer down (into `core`, next to `repositories`, ...) and rename it to `actions`.